ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewController.swift

```diff
@@ -283,15 +285,28 @@ extension KeymanWebViewController {
     do {
       data = try JSONSerialization.data(withJSONObject: stub, options: [])
     } catch {
-      log.error("Failed to serialize keyboard stub: \(error)")
+      let event = Sentry.Event(error: error)
+      event.message = SentryMessage(formatted: "Failed to serialize keyboard stub: \(error)")
+      event.extra = [:]
+      event.extra!["id"] = stub["KI"]
+      event.extra!["package"] = stub["KP"]
+
+      SentryManager.captureAndLog(event)
       return
     }
     guard let stubString = String(data: data, encoding: .utf8) else {
-      log.error("Failed to create stub string")
+      let event = Sentry.Event(level: .error)
+      event.message = SentryMessage(formatted: "Failed to create keyboard stub string for embedded KMW")
+      event.extra = [:]
+      event.extra!["id"] = stub["KI"]
+      event.extra!["package"] = stub["KP"]
+
+      SentryManager.captureAndLog(event)
       return
     }
 
-    log.debug("Keyboard stub: \(stubString)")
+    SentryManager.breadcrumbAndLog("Keyboard stub built for \(keyboard.id)", logLevel: XCGLogger.Level.none)
+    log.info("Keyboard stub: \(stubString)")
```

Comment:
Why not concatenate with breadcrumbAndLog above? (same for l. 347)

Reply:
Here, `log.info` is printing the _whole stub_, not just the keyboard's ID.  I'm not sure that we should Sentry-log the entire stub.